ci_framework/roles/dlrn_promote/tasks/check_for_previous_promotions.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Check reports from DLRN
+  ansible.builtin.shell:
```

Comment:
guess we can't use the ci_script due to zuul not exposing custom plugins?

Reply:
Yes, correct!